frame/clearing-house/src/tests/update_funding.rs

```diff
@@ -12,17 +7,25 @@ use crate::{
 			Vamm as VammPallet, MINIMUM_PERIOD_SECONDS,
 		},
 	},
-	tests::{get_market_fee_pool, set_oracle_twap},
+	tests::{
+		any_balance, any_price, get_market, get_market_fee_pool, run_for_seconds, run_to_time,
+		set_fee_pool_depth, set_maximum_oracle_mark_divergence, set_maximum_twap_divergence,
+		set_oracle_price, set_oracle_twap, traders_in_one_market_context, with_market_context,
+		with_trading_context, Balance, Market, MarketConfig, Position,
+	},
 	Direction, Error, Event,
 };
 use composable_traits::{
 	clearing_house::{ClearingHouse, Instruments},
 	time::{DurationSeconds, ONE_HOUR},
+	vamm::{AssetType, Vamm},
 };
 use frame_support::{assert_noop, assert_ok};
 use proptest::prelude::*;
 use sp_runtime::{FixedI128, FixedU128};
 
+use super::as_balance;
```

Comment:
Why not include `as_balance` in the `use crate::tests{...}` block?

(I think maybe what happened was that your IDE imported it automatically and placed the import here.)

Reply:
Yeah, that's correct. Good catch